sbomtool/go.mod

```
@@ -0,0 +1,216 @@
+module github.com/bottlerocket-os/twoliter/tools/sbomtool/go
```

Should this be the sdk?

```suggestion
module github.com/bottlerocket-os/bottlerocket-sdk/sbomtool
```